Entity/BaseFragment.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ArticleBundle\Entity;
+
+use Sonata\ArticleBundle\Model\Fragment;
+
+/**
+ * The class stores fragment information.
+ *
+ * @author Benoit Mazière <bmaziere@ekino.com>
+ */
+abstract class BaseFragment extends Fragment
+{
+    /**
+     * Updates dates before creating/updating entity.
+     */
+    public function prePersist()
```

Comment:
Is this method called by doctrine through configuration?

Maybe this should be renamed to be more comprehensive of what it is done and should be marked `final` IMHO.

Same for `preUpdate` and equivalent for `BaseArticle`.